products/workers/src/content/runtime-apis/durable-objects.md

```diff
@@ -118,6 +114,11 @@ export class Example {
   }
 }
 ```
+### In-memory state
+
+A Durable Object may be evicted from memory any time, causing a loss of all transient (in-memory) state.  To persistently store state your Durable Object might need in the future, use the Transactional Storage API.
```

Comment:
Can [Transactional Storage API] have a link out to the place in documentation where it shows how to do this?.

Reply:
It's the next line, that was why I didn't have a link.